mylyn.builds/org.eclipse.mylyn.jenkins.ui/about.html

```diff
@@ -23,5 +23,8 @@ <h3>License</h3>
 indicated below, the terms and conditions of the EPL still apply to any source code in the Content
 and such source code may be obtained at <a href="/">http://www.eclipse.org</a>.</p>
 
+<p>
+Jenkins logo from <a href="https://jenkins.io">https://jenkins.io</a>    
```

Comment:
Ah, I missed it on the first viewing
 @BeckerFrank perhaps you can find and satisfy similar requirements for GitLab logo.